static/include/services/apis/overrides/protos/vision.GetProperties.md

```diff
@@ -0,0 +1 @@
+Fetch information about what vision methods the vision service provides.
```

Comment:
See comment above; sorry I should've made that comment here instead of there since this is where we manually change it